docs/gh_pages/docs/llama_index_safe_reader.md

```diff
@@ -0,0 +1,61 @@
+# Pebblo Safe DataReader for LlamaIndex
+
+This document describes how to augment your existing Llama DocumentReader with Pebblo Safe DocumentReader to get deep data visibility on the types of Topics and Entities ingested into the Gen-AI Llama application. For details on `Pebblo Daemon` see this [pebblo server](daemon.md) document.
```

Comment:
Change `Llama application` to `LlamaIndex application`. Pls do the same for the rest of this PR.